src/.nuget/dir.props

```diff
@@ -23,7 +23,7 @@
 
     <RuntimeOS Condition="'$(RuntimeOS)' == ''">$(OSRid)</RuntimeOS>
 
-    <SupportedPackageOSGroups Condition="'$(SupportedPackageOSGroups)' == ''">Windows_NT;OSX;Linux</SupportedPackageOSGroups>
+    <SupportedPackageOSGroups Condition="'$(SupportedPackageOSGroups)' == ''">Windows_NT;OSX;Android;Linux</SupportedPackageOSGroups>
```

Comment:
@gkhanna79 The test hack I added to build.sh generates __DistroRid for Android build and, with following commits, allows package build for Android.

I've added a TODO for figuring out a reliable way to generate __DistroRid for Android. Since I generated the commit changes from the `runtime.OS._filename_.props files and guesswork, a once-over to make sure I haven't missed anything would be appreciated